src/components/MapEmbed.js

```diff
@@ -18,15 +21,16 @@ import { isSinglePostInEditor } from '../helpers/globals';
  * @return {JSX.Element} The rendered React component.
  */
 const MapEmbed = (props) => {
+	const isAdmin = select('core')?.canUser('create', 'posts');
 	const { zoom, type, className } = props;
 	let { location, height } = props;
 
 	if (!height) {
 		height = 300;
 	}
 
-	if (!isSinglePostInEditor() && !location) {
-		location = '60 29th Street #343, San Francisco, CA 94110, USA';
+	if (isAdmin && !location) {
```

Comment:
@stephenerdelyi we want to make sure this only happens in FSE, widget areas, etc and not on a single post edit because in single post edit it's expected someone would set the actual address and not need a default. Thx!

Reply:
Hey @mauteri, just pushed an update to this whenever you have a moment to check it out. It now checks to see if it's on the post editor (as opposed to other locations, like FSE/widgets, etc).

Let me know if you're having any issues or have other questions/thoughts

Reply:
Hey @stephenerdelyi mind syncing this PR? Thanks!

Reply:
@mauteri syncing now!